lib/helpers/sanitize.js

```diff
@@ -23,6 +23,10 @@ let sanitizeString = function (str) {
  * @param {Object} caps  Selenium capabilities
  */
 let caps = function (caps) {
+    if (!caps) {
```

Comment:
without input validation it would fail in line 35 (since it assumes caps is an object)